hass_nabucasa/auth.py

```diff
@@ -31,6 +31,21 @@ class Unauthenticated(CloudError):
     """Raised when authentication failed."""
 
 
+class MFARequired(CloudError):
+    """Raised when MFA is required."""
+
+    session_tokens: dict[str, str]
```

Comment:
Are these the same as you call `mfa_tokens` later? if so it should be the same here.

Reply:
Thanks for the suggestions. I improved the error class and renamed this to `mfa_tokens` as it indeed makes more sense. Also, I changed the type to `dict[str, Any]` because, although the underlying library does not set type, boto3 returns a dictionary where values can also be dictionaries or numbers.

Reply:
Oh fun 🙃